GSoC-2021/Kush-Kothari.md

```diff
@@ -0,0 +1,82 @@
+# GSoC 2021 Expression of Interest
+## Kush Kothari (Web and Python developer | ML Enthusiast)
+#### Email: kushkothari2001@gmail.com
+#### GitHub profile: [kkothari2001](https://github.com/kkothari2001)
+#### LinkedIn profile: [Kush Kothari](https://www.linkedin.com/in/kush-kothari-ba013218b/)
+
+
+## Assessment Step 1.
+
+Link to the deployed site: [ohuploaderkushk](https://ohuploaderkushk.herokuapp.com/)
+
+Link to activity page on OpenHumans: [GSoC assessment for Kush Kothari](https://www.openhumans.org/activity/gsoc-assesment-for-kush-kothari/)
+
+## Assessment Step 2.
+
+### Django Projects:
+- [To-Do App](https://github.com/kkothari2001/django-todo): App that can be used to maintain a list of tasks that can be dynamically added and deleted once completed.
+- [URL Shortner](https://github.com/kkothari2001/django-url-shortner): A bitly clone that stores URLs in a database as a hash and redirects when the URL is clicked.
+
+Please Note: While these projects show my Django skills, I believe they might not be the best way to show some of my other web development skills like web design. I urge you to go through the following links which are a showcase of some of the best projects I have made.
+
+- [The SRA Website](https://sravjti.tech/): This is the website for the Society of Robotics and Automation, VJTI. I am a member of it and have made the entire site using HTML,Sass(CSS) and JavaScript. The code is curently private (because we are working on a backend authentication system right now). But if you wish, I can provide a bundled together zip archive of all the front end mockups in HTML,Sass and JS.
+
+- [twAIn](https://github.com/kkothari2001/twAIn): As the punny name suggests, we created an AI in python that can write awesome stories after being trained on many modern classics! For this project I created the Web UI, and worked on the GPT-2 model that produces the stories.
```

Comment:
This looks really fun!! Do you have a deployment that I can play with? Don't worry if that's a lot of work / will cost too much to serve.

Reply:
Since this was a hack-athon project we too tried to deploy it somewhere, however the nature of AI projects makes the model involved HUGE. So we couldn't seem to find cost efficient solutions for it. Apologies for the lack of deployment.

However, this we did submit a video demo for it, feel free to check it out
https://drive.google.com/drive/folders/157-9lOdn5lImO3KYgIIfoL04XEfkUpTj?usp=sharing

We had lots and lots of fun exploring and picking books to train the above model on.

Reply:
https://www.youtube.com/watch?v=t4ySo3UkcYo

Feel free to check out another cool python project I made! 😊